src/components/Navbar/MobileSidebar/Layout/index.tsx

```diff
@@ -38,7 +44,24 @@ export default function NavbarMobileSidebarLayout({
       <div className={clsx('navbar-sidebar__items')}>
         <div className="navbar-sidebar__item menu">{secondaryMenu}</div>
       </div>
-      <div className={styles.sidebarFooter}>
+      <div
+        className={clsx(
+          items.length > 1 && styles.sidebarBiggerFooter,
+          styles.sidebarFooter
+        )}>
+        <div>
+          {items.length > 1 &&
+            items.map((item) => {
+              return (
+                <NavbarNavLink
+                  className={styles.sidebarLinks}
+                  to={item.to}
+                  label={item.label}
+                  key={item.label}
+                />
+              );
+            })}
```

Comment:
```suggestion
            items.map((item) => (
                <NavbarNavLink
                  className={styles.sidebarLinks}
                  to={item.to}
                  label={item.label}
                  key={item.label}
                />
              );
            )}
```
we can make it a bit shorter